modules/ROOT/pages/install-multiple-instances.adoc

```diff
@@ -2,14 +2,12 @@
 
 Installing multiple instances of Anypoint Runtime Fabric enables you to share the same cluster among multiple Runtime Fabrics, which helps you to use resources efficiently.
 
-[NOTE]
-To use this feature, you must upgrade to the minimum version of 2.2.5 of the Runtime Fabric agent that supports multiple instances.
-
-[NOTE]
-There will be 1:1 mapping from the agent namespace to the app namespace. Application namespaces cannot be shared amongst multiple instances of Runtime Fabric on the same cluster.
+* To use this feature, you must upgrade to the minimum version of 2.2.5 of the Runtime Fabric agent that supports multiple instances.
+* There will be 1:1 mapping from the agent namespace to the app namespace. Application namespaces cannot be shared amongst multiple instances of Runtime Fabric on the same cluster.
+* Hazelcast clustering is not yet supported with multiple instances.
 
 [NOTE]
-Hazelcast clustering is not yet supported with Multiple Instances.
+For multiple instances, the first Runtime Fabric agent installation creates crd for persistencegateways.rtf.mulesoft.com and priorityclass rtf-components-high-priority resources. Though is not responsible for cleanup of these resources in case of uninstalling the Runtime Fabric agent.
```

Comment:
Any font changes here?
Should "priorityclass" be two words?

Runtime Fabric agent doesn't clean up these resources when you uninstall the agent.